packages/editor-ui/src/views/WorkflowsView.vue

```diff
@@ -182,7 +206,14 @@ export default mixins(showMessage, debounceHelper).extend({
 			});
 		},
 		goToTemplates() {
-			this.$router.push({ name: VIEWS.TEMPLATES });
+			if (this.isDemoTest) {
+				this.$router.push('/collections/7');
```

Comment:
I'd prefer to use the Vue route notation and constants for this:
`this.$router.push({ name: VIEWS.COLLECTION, params: { id: '7' } });`